access-control.yaml

```
@@ -1,25 +1,57 @@
 organization: cncf-tags
 repositories:
+  - name: container-device-interface
+    external_collaborators:
+    teams:
+      cod-resource-management: admin
+  - name: wg-artefacts
```

```suggestion 
  - name: wg-artifacts
```
ref: https://github.com/cncf-tags/wg-artifacts